website/cue/reference/releases/0.36.0.cue

```diff
@@ -0,0 +1,369 @@
+
+				"""
+			contributors: ["Sinjo"]
+		},
+		{
+			type: "feat"
+			description: """
+				The `clickhouse` sink now supports `format`. This can be used to specify the data [format](https://clickhouse.com/docs/en/interfaces/formats) provided to `INSERT`s. The default is `JSONEachRow`.
+
+				"""
+			contributors: ["gabriel376"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed an issue where the `aws_s3` sink adds a trailing period to the s3 key when the `filename_extension` is empty.
+				"""
+		},
+		{
+			type: "fix"
+			description: """
+				Removed warnings for unused outputs in `datadog_agent` source when the corresponding output is disabled in the source config.
+				"""
+		},
+		{
+			type: "enhancement"
+			description: """
+				Unit tests can now populate event metadata with the `% = ...` syntax.
+
+				"""
+			contributors: ["GreyTeardrop"]
+		},
+		{
+			type: "fix"
+			description: """
+				When terminating idle HTTP connections using the configured `max_connection_age`, only send
+				`Connection: Close` for HTTP/0.9, HTTP/1.0, and HTTP/1.1 requests. This header is not supported on
+				HTTP/2 and HTTP/3 requests. This may be supported on these HTTP versions in the future.
+				"""
+		},
+		{
+			type: "feat"
+			description: """
+				Added a configuration option for the `aws_s3` source that prevents deletion of messages which failed to be delivered to a sink.
+
+				"""
+			contributors: ["tanushri-sundar"]
+		},
+		{
+			type: "fix"
+			description: """
+				The following metrics now correctly have the `component_kind`, `component_type`, and `component_id` tags:
+				    - `component_errors_total`
+				    - `component_discarded_events_total`
+
+				For the following sinks:
+				    - `splunk_hec`
+				    - `clickhouse`
+				    - `loki`
+				    - `redis`
+				    - `azure_blob`
+				    - `azure_monitor_logs`
+				    - `webhdfs`
+				    - `appsignal`
+				    - `amqp`
+				    - `aws_kinesis`
+				    - `statsd`
+				    - `honeycomb`
+				    - `gcp_stackdriver_metrics`
+				    - `gcs_chronicle_unstructured`
+				    - `gcp_stackdriver_logs`
+				    - `gcp_pubsub`
+				    - `gcp_cloud_storage`
+				    - `nats`
+				    - `http`
+				    - `kafka`
+				    - `new_relic`
+				    - `datadog_metrics`
+				    - `datadog_traces`
+				    - `datadog_events`
+				    - `databend`
+				    - `prometheus_remote_write`
+				    - `pulsar`
+				    - `aws_s3`
+				    - `aws_sqs`
+				    - `aws_sns`
+				    - `elasticsearch`
+				"""
+		},
+		{
+			type: "enhancement"
+			description: """
+				Added support for parsing HTTPS (type 65) and SVCB (type 64) resource records from DNS messages
+				"""
+			contributors: ["@esensar"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed an issue where the `journald` source was not correctly emitting metadata when `log_namespace =
+				True`.
+
+				"""
+			contributors: ["@dalegaard"]
```

Comment:
Couple of "@" slipped through it seems...

Reply:
Ah, yes indeed. I'll drop them manually.

Reply:
Dropped in [0c263a9](https://github.com/vectordotdev/vector/pull/19805/commits/0c263a978fe2572653731513964975957d5c91a8)